README.md

```diff
@@ -14,7 +14,15 @@ Clone this repository, then run:
 $ yarn dev
 ```
 
-## Deploy to Github Pages
+## Notes
+
+### [Animate CSS][1]
+
+* This demo uses additional CSS classes for animations, like `fadeIn`, which are from [`tailwindcss-animatecss`](https://github.com/bentzibentz/tailwindcss-animate.css), a [Tailwind CSS plugin](https://tailwindcss.com/docs/plugins) that integrates [Animate CSS][1].
+
+[1]: https://github.com/animate-css/animate.css)
```

Comment:
Please remove the extra `)` at the end of the link.

Reply:
🤦  Thanks for catching that